language-plutus-core/stdlib/Language/PlutusCore/StdLib/Meta/Data/Tuple.hs

```diff
@@ -51,7 +50,7 @@ tupleTypeTermAt
     :: MonadQuote m => ann -> Int -> Tuple ann -> m (Type TyName ann, Term TyName Name ann)
 tupleTypeTermAt ann ind (Tuple elTys term) = liftQuote $ do
     args <- ifor elTys $ \i ty -> do
-        n <- freshName ann $ strToBs $ "arg_" ++ show i
+        n <- freshName ann $ "arg_" <> (T.pack $ show i)
```

Comment:
Perhaps adding something like `showText ::  Eq a => a -> Text` to our prelude would be nice, but that's not too important.